pyaedt/edb_core/stackup.py

```diff
@@ -10,14 +10,14 @@
     from System import Double
     from System.Collections.Generic import List
 except ImportError:
-    warnings.warn('This module requires pythonnet.')
+    warnings.warn('This module requires Python.NET.')
```

Comment:
```suggestion
    warnings.warn('This module requires `pythonnet`.')
```
Same comment as above.